zarr.qmd

```diff
@@ -0,0 +1,68 @@
+---
+title: "Zarr"
+subtitle: Chunked and Compressed N-Dimensional Arrays
+---
+
+# What is Zarr?
+
+[Zarr](https://zarr.dev/), despite it's name, is not a scary format. It's designed for data that is too big for users' machines, but Zarr makes data small and organizes it in a way where users can take just the bits they need or distribute the load of processing lots of those bits (stored as chunks) across many machines.
+
+The Zarr data format is a community-maintained format for large-scale n-dimensional data. A Zarr store consists of compressed and chunked n-dimensional arrays. Zarr's flexible indexing and compatibility with object storage lends itself to parallel processing.
+
+Read more in the official tutorial: [Zarr Tutorial](https://zarr.readthedocs.io/en/stable/tutorial.html)
+
+## Zarr Data Organization
+
+### Arrays
+
+Zarr arrays are similar to numpy arrays, but chunked and compressed. More details about chunking and compression are to follow.
+
+### Groups
+
+Zarr supports hierarchical organization via groups.
+
+### Shape
+
+Each Zarr array has a shape which corresponds to hits length in each of its dimensions.
+
+### Coordinates and Indexes
+
+Zarr indexing supports array subsetting (both reading and writing) without loading the whole array into memory. Advanced indexing operations, such as block indexing, are detailed in the Zarr tutorial: [Advanced indexing](https://zarr.readthedocs.io/en/stable/tutorial.html#advanced-indexing).
```

Comment:
That's a good point. I feel mentioning coordinates and indexes is important in understand how you can subset data and since the Zarr tutorial itself uses python I feel ok in making this content python-specific. But I am going to add a note to capture this point.